### PR TITLE
TACO-57 - Added handling Promise

### PR DESCRIPTION
**Pull request by domjanik**

### CODE REVIEW

File: src/communication/event-types.ts [hunk 1]
```diff
@@ -189,6 +189,10 @@ export const eventsRepository: Dictionary<EventOptions> = {
 		category: '[IdentityEngine]',
```
Comment (szymon-nawrocki) [inline src/communication/event-types.ts:189]: This event can be removed

File: src/ad-services/identity-setup/index.ts [hunk 2]
```diff
@@ -1,43 +1,33 @@
...
-		communicationService.on(eventsRepository.IDENTITY_PARTNER_DATA_OBTAINED, () => {
-			const over18 = window.fandomContext.tracking.over_18;
+		return window.ie.isReady.then(() => {
+			const ppid = globalContextService.getValue('tracking', 'ppid');
+			if (ppid) {
+				targetingService.set('ppid', ppid);
+			}
 
-			if (over18) {
-				targetingService.set('over_18', over18);
```
Comment (szymon-nawrocki) [inline src/ad-services/identity-setup/index.ts:39]: Don't we want to set up `over_18` no more?

File: src/platforms/ucp-desktop/ucp-desktop-platform.ts [hunk 3]
```diff
@@ -21,7 +21,6 @@ import {
 	conditional,
 	context,
 	eventsRepository,
-	IdentitySetup,
```
Comment (szymon-nawrocki) [inline src/platforms/ucp-desktop/ucp-desktop-platform.ts:24]: Should be also removed from `ucp-mobile-platform.ts`

File: src/platforms/ucp-desktop/modes/ucp-desktop-ads.mode.ts [hunk 4]
```diff
@@ -7,13 +7,15 @@ import {
...
 	Confiant,
+	CoppaSetup,
 	DiProcess,
 	DoubleVerify,
 	DurationMedia,
 	eventsRepository,
 	Eyeota,
 	IasPublisherOptimization,
 	IdentityHub,
+	IdentitySetup,
```
Comment (szymon-nawrocki) [inline src/platforms/ucp-desktop/modes/ucp-desktop-ads.mode.ts:18]: Same logic should be applied to `ucp-mobile-ads.mode.ts`

File: src/platforms/shared/bootstrap.ts [hunk 5]
```diff
@@ -1,12 +1,23 @@
-import { logVersion, utils } from '@wikia/ad-engine';
-import { trackingOptInWrapper } from './consent/tracking-opt-in-wrapper';
```
Comment (szymon-nawrocki) [inline src/platforms/shared/bootstrap.ts:2]: `TrackingOptInWrapper` class is not used anymore, so it looks like it should be removed

**Review by szymon-nawrocki — CHANGES_REQUESTED** [review]
Remove old not used code, and apply this changes to ucp-mobile platform

File: src/ad-services/identity-setup/index.ts (hunk 2 above)
Comment (domjanik) [reply src/ad-services/identity-setup/index.ts:39]: Unfortunately - no :( Whole Pipeline should be stopped long time ago

**Review by szymon-nawrocki — DISMISSED** [review]

**Pull request closed without merging**